mantidproject/mantid
# Update nonorthog plot with correct limits in sliceviewer

**Description of work.**

Fixed bug whereby plot was getting updated with the orthog limits in nonorthog mode (but not in new plot hence clicking the home button corrected the plot).

**To test:**

(1) Follow instructions in original issue
(2) Check limits and zooming etc. work on orthogonal workspaces (and non-orthog in orthog mode) and matrix workspaces

Can create event and histo workspaces like so
```
ws = CreateMDWorkspace(Dimensions='3', Extents='-6,6,-4,4,-0.5,0.5',
                       Names='H,K,L', Units='r.l.u.,r.l.u.,r.l.u.',
                       Frames='HKL,HKL,HKL',
                       SplitInto='2', SplitThreshold='50')

expt_info = CreateSampleWorkspace()
ws.addExperimentInfo(expt_info)
SetUB(ws, 1,1,2,90,90,120)
FakeMDEventData(ws, PeakParams='1e+06,2,0,0,1', RandomSeed='3873875')
 
# try with MDHistoToWorkspace
BinMD(InputWorkspace='ws', AxisAligned=False, 
    BasisVector0='H,r.l.u.,1.0,0.0,0.0', 
    BasisVector1='K,r.l.u.,0.0,1.0,0.0', 
    BasisVector2='L,r.l.u.,0.0,0.0,1.0',
    OutputExtents='-6,6,-4,4,-0.5,0.5', OutputBins='100,100,100', 
    NormalizeBasisVectors=False, OutputWorkspace='ws_histo')
```

Fixes #32317 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
